### ISSUE TITLE
Спасибо отличный редактор много функций ! Целый комплекс совмещенный с CMS ! 

### ISSUE THREAD

**Issue by aukc1970** [hollpee-builder/builder#1]
Добрый день ! Спасибо отличный редактор много функций ! Целый комплекс совмещенный с CMS !

**Comment by Ksenof** [comment]
Добрый день, у меня тоже не корректно работает выгрузка в CMS. Подскажите пож.ст как вы решили этот вопрос?

**Comment by aukc1970** [comment]
Я думаю что это зависит от локального веб сервера который Вы используете, я стал использовать Wampserver и сделал все как описано в инструкции и все работает !

**Comment by Ksenof** [comment]
Нет, сам сервис у меня работает на Wampserver. Все отлично. После выгрузки проекта, заливаю CMS Маркетолог на хостинг, при заходе на сайт перебрасывает на вкладку sait.ru/admin/admin/ и там путая страница. Вы не сталкивались с этим?

**Comment by aukc1970** [comment]
У меня после заливки на хостинг появляется страница где можно создать администратора попробуйте залить cms на локальный веб сервер например Wampserver еще есть видео вот ссылки https://www.youtube.com/watch?v=2Lu6K9x-jQo или https://www.youtube.com/channel/UCk1rZAyhM5hHNPayrsl4CLA/videos

**Comment by Ksenof** [comment]
Для теста собрал из базовых блоков. Если есть возможность посмотрите пож.ст https://yadi.sk/d/oedFv0UfbzskXg установится у вас или нет? У меня та же проблема, просто белый  экран. Спасибо!

**Comment by aukc1970** [comment]
Вы почему то скрываете о себе информацию. Заливать на хостинг файлы от неизвестного источника не безопасно. Если Вам действительно нужна помощь напишите мне с реального Вашего адреса в ответ я Вам напишу что Вам можно попробовать сделать.

**Comment by Ksenof** [comment]
Напишите контакты куда вам написать? Можем даже голосом пообщаться.

**Comment by aukc1970** [comment]
aukc1970@yandex.ru напишите со своей почты потом можно будет общаться по watsap.
